contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -388,6 +389,17 @@ check_tsql_version(char **newval, void **extra, GucSource source)
 	return true;
 }
 
+static bool
+check_endpoint(char **newval, void **extra, GucSource source)
```

Comment:
Is this required now? A PG guc cannot be set from TDS endpoint using set command

Reply:
removed the check